maintainers/maintainer-list.nix

```diff
@@ -2468,6 +2468,12 @@
     githubId = 50839;
     name = "Brian Jones";
   };
+  bonsairobo = {
+    name = "Duncan Fairbanks";
+    email = "duncanfairbanks6@gmail.com";
+    github = "bonsairobo";
+    githubId = 3229981;
+  };
```

Comment:
Adding yourself to maintainers needs to be a separate commit: `maintainers: add bonsairobo`